apns_csr/builder.py

```diff
@@ -14,7 +14,7 @@
 def _get_remote_cert(url):
 # TODO(dustin): There's an encoding problem with requests.get(url).text, and partial data when we do requests.get(url).raw.read() .
 # TODO(dustin): Cache/store this locally, or we'll be out of luck if their systems are unavailable.
-    data = urllib2.urlopen(url).read()
+    data = urllib.request.urlopen(url).read()
```

Comment:
This breaks Python2. Please try to import urllib2, first, and, if it imports, use what we had. If it doesn't import, try importing this and, if it imports, use your new logic.